iac-extensions/docker/src/test/java/org/sonar/iac/docker/tree/impl/ExecFormImplTest.java

```diff
@@ -30,10 +33,6 @@
 import org.sonar.iac.docker.tree.api.ExecForm;
 import org.sonar.iac.docker.tree.api.SyntaxToken;
 
-import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
static imports should be here not before other imports 
https://github.com/SonarSource/sonar-developer-toolset

Reply:
I agree, I don't get why my intellij keep reorganizing them before :confused:

![image](https://github.com/user-attachments/assets/68f5fee3-7833-42c4-a27e-4fbd1df5be8c)

I tried several things including installing the last version, I'm unable to get it to properly organize the imports :expressionless:

Reply:
Finally found a way by starting on a fresh **sonar-iac** project.